instance/models/mixins/openedx_database.py

```diff
@@ -214,9 +217,26 @@ def generate_var_name(var):
             generate_var_name("pass"): self._get_mysql_pass(user),
         }
 
+    @staticmethod
+    def new_rabbitmq_user():
+        """
+        Return a new RabbitMQ user.
+        """
+        user = None
+        while user is None:
```

Comment:
I hate to harp on it, but we need to consider the degenerate case - let's limit the number of creation attempts, even though it realistically shouldn't make a difference.